app/assets/stylesheets/text.scss

```diff
@@ -1,5 +1,5 @@
 @import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
-.govuk-heading-xl, .govuk-heading-l, .govuk-heading-m, .govuk-body, .govuk-body-s, .govuk-link {
+.govuk-heading-xl, .govuk-heading-l, .govuk-heading-m, .govuk-body, .govuk-body-s, .govuk-link, .govuk-button {
```

Comment:
We can probably just apply this to the body now? :thinking:

Reply:
Unfortunately applying this to `body` does not work as the `govuk-frontend` overrides font-family for each heading class.